procedures/foreman/branch.md.erb

```diff
@@ -36,27 +36,25 @@
   * [ ] Sync the community-templates to Foreman core, by running `script/sync_templates.sh`
   * [ ] Create a <%= @release %>-stable branch in [community-templates](https://github.com/theforeman/community-templates)
 * [ ] Add new languages that are at a [reasonable completion on Transifex](https://www.transifex.com/foreman/foreman/foreman/) to **develop**
-* [ ] [Generate](https://github.com/ekohl/theforeman-rel-eng/blob/master/generate_gpg), [backup](https://github.com/ekohl/theforeman-rel-eng/blob/master/export_gpg_private), [sign](https://github.com/ekohl/theforeman-rel-eng/blob/master/sign_gpg) and [upload](https://github.com/ekohl/theforeman-rel-eng/blob/master/upload_gpg) a GPG key. See [GPG_Keys](https://projects.theforeman.org/projects/foreman/wiki/GPG_Keys) if needed.
-* [ ] Publish the key by [exporting](https://github.com/ekohl/theforeman-rel-eng/blob/master/export_gpg_public) the GPG key
+* [ ] Create new settings files for [foreman](https://github.com/theforeman/theforeman-rel-eng/blob/master/releases/foreman/<%= release %>/settings) and [client](https://github.com/theforeman/theforeman-rel-eng/blob/master/releases/client/<%= release %>/settings), ensure `client` has the rights `OSES` list.
+* [ ] [Generate](https://github.com/theforeman/theforeman-rel-eng/blob/master/generate_gpg), [backup](https://github.com/theforeman/theforeman-rel-eng/blob/master/export_gpg_private), [sign](https://github.com/theforeman/theforeman-rel-eng/blob/master/sign_gpg) and [upload](https://github.com/theforeman/theforeman-rel-eng/blob/master/upload_gpg) a GPG key. See [GPG_Keys](https://projects.theforeman.org/projects/foreman/wiki/GPG_Keys) if needed.
+* [ ] Publish the key by [exporting](https://github.com/theforeman/theforeman-rel-eng/blob/master/export_gpg_public) the GPG key
   * [ ] Update the website's [security.md](https://github.com/theforeman/theforeman.org/blob/gh-pages/security.md) and create a file in [static/keys](https://github.com/theforeman/theforeman.org/tree/gh-pages/static/keys)
   * [ ] Create [releases/<%= release %>/RPM-GPG-KEY-foreman](https://yum.theforeman.org/releases/<%= release %>/RPM-GPG-KEY-foreman) on yum.theforeman.org
+  * [ ] Add key to the setting files for [foreman](https://github.com/theforeman/theforeman-rel-eng/blob/master/releases/foreman/<%= release %>/settings) and [client](https://github.com/theforeman/theforeman-rel-eng/blob/master/releases/client/<%= release %>/settings)
```

Comment:
This is unclear to me. `export_gpg_private` should already refuse to operate if `FULLGPGKEY` is not set. `generate_gpg` should already instruct you to set `FULLGPGKEY` in the settings file and maybe it can even be automated. https://github.com/theforeman/theforeman-rel-eng/pull/19 should do that.